examples/observers/ollama_example.py

```diff
@@ -4,6 +4,9 @@
 # Ollama is running locally at http://localhost:11434/v1
 openai_client = OpenAI(
     base_url="http://localhost:11434/v1",
+    # The openai client needs an api key to instantiate, but we
```

Comment:
can your evert these changes? I think people are normally aware of this.

Reply:
The change was to allow it to run without exporting one (the bit that sets it to unused) but if it's a norm not to worry about it, i'm not attached to this change